src/java.base/share/classes/java/lang/ModuleLayer.java

```diff
@@ -498,7 +501,7 @@ public ModuleLayer defineModules(Configuration cf,
      */
```

Comment:
I had wondered if `@Nullable` might make sense for the `ClassLoader` type argument of the `Function` in `defineModules(Configuration, Function)` above, since this method declares methods whose `ClassLoader` parameters are `@Nullable`. But this one delegates to the other `defaultModules` overload, which is explicit about rejecting null results from the `Function`.